CHANGELOG.txt

```diff
@@ -23,6 +23,11 @@
 * CMS handles uninstalled apphooks better
 * Added the ability to make pagetree actions work on currently picked language
 * Removed deprecated CMS_TOOLBAR_SIMPLE_STRUCTURE_MODE setting
+* Introduction of the method `get_cache_expiration` on CMSPluginBase to be used
+  by plugins for declaring their rendered content's period of validity.
+* Introduction of the method `get_vary_cache_on` on CMSPluginBase to be used
+  by plugins for declaring ``VARY`` headers.
+* Substantial improvements to the placeholder cache
```

Comment:
Wherever it makes sense, I would like all lines in the changelog to start with a verb - _added_, _removed_, _improved_, _fixed_, _introduced_ etc.I think it makes it easier to scan and understand what has been done.